celery/app/base.py

```diff
@@ -471,6 +477,18 @@ def run(*args, **kwargs):
                     try:
                         return task._orig_run(*args, **kwargs)
                     except autoretry_for as exc:
+                        if retry_backoff:
```

Comment:
> `random.randrange` excludes the endpoint that is why I added `1.` This was also excluding defining `1` as a possible jitter value.

Good catch! Thanks, I fixed that.

As for making the code more compact, I'd like to do that, but I'm bumping up against the "max characters per line" style restriction. So, I'll keep things the way they currently are in the pull request.